### PR TITLE
OPS-8245: Update base image and installed chrome version

### PR DESCRIPTION
**Pull request by berliner**

### CODE REVIEW

**Review by teodorescuserban — APPROVED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -28,9 +28,11 @@ RUN \
...
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    rm -rf /tmp/* && \
+    rm -rf /tmp/*
+
+Run \
     mkdir -p /srv/www /root /var/run/s6 /etc/services.d/snap && \
-    cp docker/run_snap /etc/services.d/snap/run && \
+    cp /srv/www/run_snap /etc/services.d/snap/run && \
     rm -rf /etc/services.d/node
```
Comment (cafuego) [inline docker/Dockerfile]: This cna be part of a single step, which makes the image a bit smaller. I'm also not convinced there is a /srv/www/run_snap to copy.

**Comment by cafuego** [comment]
I was right about the init script, that build just didn't work at all. Then I got thinking...

Since the snap service always runs the stable chrome anyway and is basically the base node image plus chrome (and dependencies) plus an app, let's just remove the snap service app and drop in the service script that starts chrome.

I've built this image and pushed it as `public.ecr.aws/unocha/hpc-headless-chrome:OPS-8245` so you can test that it works.

An even easier fix would be to drop the service script in the stack repository and map an `/etc/services.d` into the tools-snap-service image, so you can outright use that unmodified. (This seems to work fine when I try it locally)

**Review by teodorescuserban — APPROVED** [review]
lgtm

**Comment by berliner** [comment]
Your change in approach makes sense, there is just one reason that would justify downloading a specific version of chrome, and that is that puppeteer and chrome are tightly bound, but run in different containers and different stacks in our setup. So keeping them in sync is probably easier with pinned versions. I'll give this some thought.

On the above. The reason to separate the build steps was only to speed up debugging on my local. And the change to `/srv/www/run_snap` was trial-and-error to get the image to build on my local.

**Comment by cafuego** [comment]
Ah yep, I'd forgotten about the puppeteer / chrome version issue. It probably still makes sense to use the snap service as base image, it already comes with all required dependencies for chrome anyway and since the image layers are shared, it'll use less storage on the servers. I'll amend this PR for that for now, so you can test and deploy.

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
 ENV WORKDIR=/srv/www
 
 LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vendor="HPC" \
       org.label-schema.version=$VERSION \
-      org.label-schema.name="hpc-headless_chrome" \
-      org.label-schema.description="This service provides a headless-chrome " \
+      org.label-schema.name="hpc-headless-chrome" \
+      org.label-schema.description="This service provides a headless-chrome for FTS" \
```
Comment (cafuego) [inline docker/Dockerfile]:
```suggestion
      org.label-schema.description="This service provides a headless-chrome for Humanitarian Insight" \
```

**Comment by cafuego** [comment]
@berliner Ok, have a look at *this* version :-)

**Comment by berliner** [comment]
That looks good to me, builds on local and works fine.

**Pull request merged**